src/SpendPermissionManager.sol

```diff
@@ -234,27 +227,15 @@ contract SpendPermissionManager is EIP712 {
         _approve(spendPermission);
     }
 
-    /// @notice Spend tokens using a spend permission, transferring them from `account` to `spender`.
-    ///
-    /// @param spendPermission Details of the spend permission.
-    /// @param value Amount of token attempting to spend.
-    function spend(SpendPermission memory spendPermission, uint160 value)
-        public
-        requireSender(spendPermission.spender)
-    {
-        _useSpendPermission(spendPermission, value);
-        _transferFrom(spendPermission.token, spendPermission.account, spendPermission.spender, value);
-    }
-
-    /// @notice Approve a spend permission batch via a signature from the account, optionally deploying account.
+    /// @notice Approve a spend permission batch via a signature from the account.
     ///
-    /// @dev Compatible with ERC-6492 signatures (https://eips.ethereum.org/EIPS/eip-6492)
-    /// Validates a signature over a batch of spend permission data and optionally deploys account.
+    /// @dev Compatible with ERC-6492 signatures including side effects (https://eips.ethereum.org/EIPS/eip-6492).
+    /// @dev Does not enforce uniqueness of permissions within a batch, allowing duplicate approvals.
```

Comment:
as a dev I might find this a little confusing because it's not clear to me whether duplicate approvals lead to two different approved permissions or whether it's a no-op.

perhaps 
```suggestion
Does not enforce uniqueness of permissions within a batch, allowing idempotent approvals.
```